### DISCUSSION TITLE
How do I turn off the Inter-roman and Inter-italic fonts?

### DISCUSSION THREAD

**Discussion by johnantoni** [ElMassimo/vite_ruby#408] [Q&A]
Hi, I'm getting two error regarding the Inter-roman & Inter-italic fonts.

I believe the problem is to do with my cors setup, but is there any way to disable these fonts from trying to load?

```
plugin:1 Refused to load the font 'http://127.0.0.1:3000/vite-dev/assets/Inter-roman.var-17fe38ab.woff2?v=3.19' because it violates the following Content Security Policy directive: "default-src 'none'". Note that 'font-src' was not explicitly set, so 'default-src' is used as a fallback.

plugin:1 Refused to load the font 'http://127.0.0.1:3000/vite-dev/assets/Inter-italic.var-d1401419.woff2?v=3.19' because it violates the following Content Security Policy directive: "default-src 'none'". Note that 'font-src' was not explicitly set, so 'default-src' is used as a fallback.
```

If that's not posssible is there a way to allow serving the vite-dev files on startup?

**Comment by ElMassimo** [comment]
Hi John!

See [this example](https://github.com/ElMassimo/vite_ruby/blob/main/examples/rails/config/initializers/content_security_policy.rb#L17) on how to configure CORS during development, allowing the browser to connect to the Vite dev server.

**Reply by johnantoni** [reply]
excellent!

thank you, this resolves the issue.
